README.md

```diff
@@ -421,6 +423,9 @@ following properties:
 
     * `reason` (String) -- A string describing the reason for an unsupported video recommendation.
       For example, `'No camera was found.'`
+    
+    * `qualityLimitationReason` (String) --  The qualityLimitationReason indicates the reason behind 
+       any limitations in media quality during a session. It can have values: `cpu` for CPU overload, `bandwidth` for insufficient network bandwidth and `none` if there is no limition.
```

Comment:
So, if you try to test FHD, and it is not supported, this is set, right? But the `recommendedResolution` could be set to a lower resolution? That may be confusing.

Reply:
We can discuss about it @jeffswartz